cloudslang_dsl_reference.rst

```diff
@@ -877,6 +882,15 @@ referenced by in the file. Using an alias is one way to reference the
             do:
               ops.print:
                 - text: "Hi"
+        - run_subflow:
+            do:
+              subs.division:
+                - input1: "5"
+                - input2: "3"
+
+In this example, the ``print`` operation's source file defines its namespace as
+``example.utils`` and the ``division`` flow's source file defines its namespace
+as ``example.subflows``.
```

Comment:
I think that the explanation is not clear enough
It should be something like: the ops alias of the print operation defines its namespace...